utils/swift_build_support/tests/products/test_ninja.py

```diff
@@ -71,20 +74,23 @@ def tearDown(self):
         self.args = None
 
     def test_ninja_bin_path(self):
-        ninja_build = Ninja(
+        ninja_build = Ninja.make_builder(
```

Comment:
Not sure I'm in love with the name `make_builder`. I'd lean more towards something like `builder` or `new_builder`. But this is all a matter of opinion.

Reply:
I will change it to `new_builder`. I tried `builder` and it didn’t feel right. The verb in front makes it more obvious (to me), that it is something new being created, instead of something part of the product.